### PR TITLE
Add contentDescription attributes for all ImageViews

### PR DESCRIPTION
**Pull request by FarbodSalamat-Zadeh**
See #166 

I have added `contentDescription` attributes to all of the `ImageView`s as requested. Please let me know if there's anything that needs to be improved.

_For GCI 2016_

### CODE REVIEW

**Comment by FarbodSalamat-Zadeh** [comment]
I've refactored some of the string names as requested.

File: PowerUp/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -46,32 +46,32 @@
...
+    <string name="left_eye_description">Left eye</string>
+    <string name="right_eye_description">Right eye</string>
+    <string name="left_hair_description">Left hair</string>
+    <string name="right_hair_description">Right hair</string>
+    <string name="left_face_description">Left face</string>
+    <string name="right_face_description">Right face</string>
+    <string name="clothes_description">Clothes</string>
+    <string name="left_clothes_description">Left clothes</string>
+    <string name="right_clothes_description">Right clothes</string>
+    <string name="health_text_description">Health text</string>
```
Comment (riddhik84) [inline PowerUp/app/src/main/res/values/strings.xml]: "Health text" value is not descriptive to understand the image when used with TalkBack.
Please change the value as
<string name="health_text_description">Health and Powers</string>

Comment (FarbodSalamat-Zadeh) [inline PowerUp/app/src/main/res/values/strings.xml]: Done.

**Comment by chhavip** [comment]
Closing as was approved in GCI

**Pull request closed without merging**